src/components/OptionsSelector/BaseOptionsSelector.js

```diff
@@ -58,7 +58,7 @@ const defaultProps = {
     referralContentType: CONST.REFERRAL_PROGRAM.CONTENT_TYPES.REFER_FRIEND,
     safeAreaPaddingBottomStyle: {},
     contentContainerStyles: [],
-    listContainerStyles: [styles.flex1],
+    listContainerStyles: [{flex: 1}],
```

Comment:
I see that listContainerStyles is being used once. So inside render we can define a variable `listContainerStyles` and assign `this.props.listContainerStyles || [styles.flex1]`. 

```suggestion
    listContainerStyles: undefined,
```